apis/Google.Cloud.DevTools.Common/Google.Cloud.DevTools.Common/Source/V1/SourceContext.File.cs

```diff
@@ -104,9 +103,5 @@ private static string GetFilePath()
             || ex is NotSupportedException
```

Comment:
Is it okay to swallow these?  I understand the not found bits but SecurityException or UnauthorizedAccessException seem useful to see.